parachains/runtimes/bridge-hubs/bridge-hub-rococo/tests/tests.rs

```diff
@@ -52,5 +62,166 @@ bridge_hub_test_utils::test_cases::include_teleports_for_native_asset_works!(
 			_ => None,
 		}
 	}),
-	1013
+	bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID
+);
+
+bridge_hub_test_utils::include_initialize_bridge_by_governance_works!(
+	initialize_bridge_to_wococo_by_governance_works,
+	Runtime,
+	BridgeGrandpaWococoInstance,
+	bridge_hub_test_utils::CollatorSessionKeys::new(
+		AccountId::from(ALICE),
+		AccountId::from(ALICE),
+		SessionKeys { aura: AuraId::from(sp_core::sr25519::Public::from_raw(ALICE)) }
+	),
+	bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+	Box::new(|call| RuntimeCall::BridgeWococoGrandpa(call).encode())
+);
+
+bridge_hub_test_utils::include_handle_export_message_from_system_parachain_to_outbound_queue_works!(
+	handle_export_message_from_system_parachain_to_outbound_queue_works_for_wococo,
+	Runtime,
+	XcmConfig,
+	WithBridgeHubWococoMessagesInstance,
+	bridge_hub_test_utils::CollatorSessionKeys::new(
+		AccountId::from(ALICE),
+		AccountId::from(ALICE),
+		SessionKeys { aura: AuraId::from(sp_core::sr25519::Public::from_raw(ALICE)) }
+	),
+	bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+	1000,
+	Box::new(|runtime_event_encoded: Vec<u8>| {
+		match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
+			Ok(RuntimeEvent::BridgeWococoMessages(event)) => Some(event),
+			_ => None,
+		}
+	}),
+	|| ExportMessage { network: Wococo, destination: X1(Parachain(1234)), xcm: Xcm(vec![]) },
+	bridge_hub_rococo_config::DEFAULT_XCM_LANE_TO_BRIDGE_HUB_WOCOCO
+);
+
+bridge_hub_test_utils::include_message_dispatch_routing_works!(
+	message_dispatch_routing_works_for_wococo_messages,
+	Runtime,
+	XcmConfig,
+	ParachainSystem,
+	WithBridgeHubWococoMessagesInstance,
+	RelayNetwork,
+	bridge_hub_rococo_config::WococoGlobalConsensusNetwork,
+	bridge_hub_test_utils::CollatorSessionKeys::new(
+		AccountId::from(ALICE),
+		AccountId::from(ALICE),
+		SessionKeys { aura: AuraId::from(sp_core::sr25519::Public::from_raw(ALICE)) }
+	),
+	bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+	1000,
+	Box::new(|runtime_event_encoded: Vec<u8>| {
+		match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
+			Ok(RuntimeEvent::ParachainSystem(event)) => Some(event),
+			_ => None,
+		}
+	}),
+	Box::new(|runtime_event_encoded: Vec<u8>| {
+		match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
+			Ok(RuntimeEvent::XcmpQueue(event)) => Some(event),
+			_ => None,
+		}
+	}),
+	bridge_hub_rococo_config::DEFAULT_XCM_LANE_TO_BRIDGE_HUB_WOCOCO
 );
+
+mod bridge_hub_wococo {
```

Comment:
This is needed only for `include_teleports_for_native_asset_works!`, but I guess it's fine to keep all wococo tests inside.

Maybe add rococo tests to `mod bridge_hub_rococo` as well for consistency?